book.toml

```diff
@@ -3,6 +3,6 @@ authors = ["Flavio Castelli"]
 language = "en"
 multilingual = false
 src = "src"
-title = "Chimera Kubernetes Policy Engine"
-git-repository-url = "https://github.com/chimera-kube/chimera-book"
-site-url = "/chimera-book"
+title = "Kubewarden Kubernetes Policy Engine"
+git-repository-url = "https://github.com/kubewarden/kubewarden-book"
+site-url = "/"
```

Comment:
Not sure about this change, I think this had a reason...

Reply:
yeah, because now we have `docs.kubewarder.io` instead of `something-github-something/chimera-book`